cp-ksql-server/include/etc/confluent/docker/launch

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# JMX settings
+if [ -z "$KSQL_JMX_OPTS" ]; then
+  KSQL_JMX_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false "
```

Comment:
Double-checking: This setup means that JMX connections are never secured (no encryption, no authentication) unless `KSQL_JMX_OPTS` is defined appropriately, even if the KSQL server was otherwise configured with security settings.

Is this ok? I think the answer is Yes, but we should then perhaps add a note to our security config section that the user should not only think about `KSQL_OPTS` for "normal" security settings, but also configure `KSQL_JMX_OPTS` appropriately.

Reply:
Do we need JMX enabled at all, by default?

Reply:
I am not sure we need JMX. But right now the KSQL server doesn't support SSL on the rest endpoint anyway. So there is no there is no discrepancy here.

Reply:
I'd probably err on the side of enabling JMX by default -- you do want to monitor your running containers, don't you?

Reply:
JMX will be enabled if the `KSQL_JMX_PORT` environment variable is defined.